packages/ariakit/src/toolbar/__examples__/toolbar/style.css

```diff
@@ -0,0 +1,36 @@
+@import url("../../../button/__examples__/button/style.css");
+
+.toolbar {
+  @apply
+    flex
+    gap-1
+    border
+    p-1
+    rounded
+    border-alpha-1
+    flex-row
+    w-fit
+    items-center
+    bg-canvas-4
+    dark:bg-canvas-4-dark
+  ;
+}
+
+.button {
+  @apply
+    bg-canvas-4
+    text-canvas-2
+    hover:bg-canvas-4-hover
+    dark:hover:bg-canvas-4-dark-hover
+    dark:bg-canvas-4-dark
+    text-sm
+  ;
+}
+
+.button svg {
+  @apply sm:block hidden;
```

Comment:
here i am removing the icon when the screen reaches the width of sm. This so as not to break the playground

Reply:
That makes sense. We can also allow horizontal scrolling on the playground preview in such cases.